include/QtNodes/internal/DataFlowGraphModel.hpp

```diff
@@ -94,17 +94,23 @@ class NODE_EDITOR_PUBLIC DataFlowGraphModel : public AbstractGraphModel, public
         return model;
     }
 
+public Q_SLOTS:
+    virtual void propagate(NodeId const nodeId);
+
+protected:
+    virtual bool canPropagate(ConnectionId const) { return true; }
```

Comment:
It should be a const method

Reply:
Good catch thank you. Apologies for the late reply, I've been very busy of late.